README.md

```diff
@@ -1,31 +1,133 @@
+ * **micropython**: MicroPython-specific packages that do not have equivalents
+     in other Python environments. This includes drivers for hardware
+     (e.g. sensors, peripherals, or displays), or libraries to work with
+     embedded functionality (e.g. bluetooth), or MicroPython-specific
+     packages that do not have equivalents in CPython.
 
- * **unix-ffi**: These modules are specifically for the MicroPython Unix port and provide access to operating-system and third-party libraries via FFI.
+* **unix-ffi**: These packages are specifically for the MicroPython Unix port
+    and provide access to operating-system and third-party libraries via FFI,
+    or functionality that is not useful for non-Unix ports.
 
-Usage
------
+## Usage
 
-Many libraries are self contained modules, and you can quickly get started by
-copying the relevant Python file to your device. For example, to add the
-`base64` library, you can directly copy `python-stdlib/base64/base64.py` to the `lib`
-directory on your device.
+To install a micropython-lib package, there are four main options. For more
+information see the [Package management documentation](https://docs.micropython.org/en/latest/reference/packages.html)
+documentation.
 
-Other libraries are packages, in which case you'll need to copy the directory instead. For example, to add `collections.defaultdict`, copy `collections/collections/__init__.py` and `collections.defaultdict/collections/defaultdict.py` to a directory named `lib/collections` on your device.
+### On a network-enabled device:
 
-Future plans (and new contributor ideas)
-----------------------------------------
+As of MicroPython v1.20 (and nightly builds since October 2022), boards
+with WiFi and Ethernet support include the `mip` package manager.
+
+```py
+>>> import mip
+>>> mip.install("package-name")
+```
+
+### Using `mpremote` from your PC:
+
+This is the officially-supported tool for interacting with a MicroPython
+device, and since v0.4.0 it supports installing packages using the `mip`
+command.
+
+```bash
+$ mpremote connect /dev/ttyUSB0 mip install package-name
+```
+
+See the [mpremote documentation](https://docs.micropython.org/en/latest/reference/mpremote.html).
+
+### Freeze into your firmware:
+
+If you are building your own firmware, all packages in this repository include
+a `manifest.py` that can be included into your board manifest via the
+`require()` command. See [Manifest files](https://docs.micropython.org/en/latest/reference/manifest.html#require) for
+more information.
+
+### Copy the files manually:
+
+Many micropython-lib packages are just single-file modules, and you can
+quickly get started by copying the relevant Python file to your device. For
+example, to add the `base64` library, you can directly copy
+`python-stdlib/base64/base64.py` to the `lib` directory on your device.
+
```

Comment:
I think an example may be warranted here, to guide people to use mpremote and show the basic case. Something like:

```bash
> mpremote connect /dev/ttyUSB0 cp python-stdlib/base64/base64.py :/lib
```

Reply:
Done.

I did initially have this, but the problem is that the package-directory case isn't supported well with mpremote (because `cp -r` can only copy to `:` not `:/lib`). I've just added the single-file example, and we can update the second case when mpremote is fixed.